packages/adminrouter/extra/src/docs/api/nginx.master.yaml

```diff
@@ -234,11 +234,11 @@ routes:
       file: conf/lib/metadata.lua
     path: /metadata
   /navstar/lashup/key:
-    group: Navstar
+    group: DC/OS Net
     matcher: path
     proxy:
       path: 'http://$backend/lashup/key'
-      backend: navstar
+      backend: dcos_net
```

Comment:
Every where else we are using `dcos-net`, can we s/dcos_net/dcos-net?

Reply:
Unfortunately, nginx doesn't support `-` symbol in literals :(